### ISSUE TITLE
Package incompatible with laravel/framework 8.62.0

### ISSUE THREAD

**Issue by mortenscheel** [jasonmccreary/laravel-test-assertions#32]
PhpUnit throws a fatal error when using this package with laravel 8.62.0 and PHP 8.0:
```
PHP Fatal error:  Declaration of JMac\Testing\Traits\AdditionalAssertions::assertNotSoftDeleted(Illuminate\Database\Eloquent\Model $model) must be compatible with Illuminate\Foundation\Testing\TestCase::assertNotSoftDeleted($table, array $data = [], $connection = null, $deletedAtColumn = 'deleted_at') in /var/www/html/tests/TestCase.php on line 182
```

This is because `assertNotSoftDeleted()` has been added to `Illuminate\Foundation\Testing\Concerns\InteractsWithDatabase` but with a different method signature.

A temporary workaround is to add the following method to the class that imports the `AdditionalAssertions` trait:
```php
protected function assertNotSoftDeleted($table, array $data = [], $connection = null, $deletedAtColumn = 'deleted_at')
{
    return parent::assertNotSoftDeleted($table, $data, $connection, $deletedAtColumn);
}
```

**Comment by jasonmccreary** [comment]
This was already reported in #30 and resolved with the 2.0.0 release.

**Comment by mortenscheel** [comment]
Ok thanks. I thought I was using the latest version.
